app/views/sentences/several_random_sentences.ctp

```diff
@@ -26,7 +26,7 @@
  */
 
 $this->set('title_for_layout', $pages->formatTitle(__('Random sentences', true)));
-
+if(!isset($searchProblem)) {
 ?>
 <div id="annexe_content">
     <div class="module">
```

Comment:
The annexe content should be left visible regardless of the error, so that users can click the button to try again.